### PR TITLE
refactor(core): workbench

### PR DESCRIPTION
**Pull request by EYHN**
Merge the right sidebar logic into the workbench. this can simplify our logic.

Previously we had 3 modules

* workbench
* right-sidebar (Control sidebar open&close)
* multi-tab-sidebar (Control tabs)

Now everything is managed in Workbench.

# Behavioral changes

The sidebar button is always visible and can be opened at any time.
If there is no content to display,  will be `No Selection`

Elements in the sidebar can now be defined as`unmountOnInactive=false`. Inactive sidebars are marked with `display: none` but not unmount, so the `ChatPanel` can always remain in the DOM and user input will be retained even if the sidebar is closed.

### CODE REVIEW

**Comment by EYHN** [comment]
* **#7355** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7355?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by pengx17 — COMMENTED** [review]
can you update the description on the purpose of changes?

File: packages/frontend/core/src/modules/workbench/entities/view.ts [hunk 1]
```diff
@@ -2,13 +2,25 @@ import { Entity, LiveData } from '@toeverything/infra';
...
 import { Observable } from 'rxjs';
 
-import { createIsland } from '../../../utils/island';
 import { createNavigableHistory } from '../../../utils/navigable-history';
 import type { ViewScope } from '../scopes/view';
 
 export class View extends Entity {
   id = this.scope.props.id;
 
+  sidebarTabs$ = new LiveData<string[]>([]);
```
Comment (CatsJuice) [inline packages/frontend/core/src/modules/workbench/entities/view.ts]: 🤔 Why not keep `Sidebar` entity but manage in view directly?

Comment (EYHN) [reply packages/frontend/core/src/modules/workbench/entities/view.ts]: I deleted the sidebar, and the sidebar is now managed by the view

Comment (CatsJuice) [reply packages/frontend/core/src/modules/workbench/entities/view.ts]: Yes, I meant that is it more intuitive to maintain a `Sidebar` entity for the logic of the sidebar, as  `view.sidebar`?

**Review by CatsJuice — APPROVED** [review]

**Comment by EYHN** [comment]
> can you update the description on the purpose of changes?

Simplified the logic of the sidebar, which was a bit over-designed before.

And now that the sidebar is included in the workbench, their states can also be saved and persisted together.

**Pull request merged**